packages/dds/tree/package.json

```diff
@@ -28,6 +28,7 @@
     "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "fence": "gf",
```

Comment:
Do we want to `&&` this into our build script above? Otherwise I'm afraid it will not be enforced.

Reply:
Interesting. If this turns out to be a good experiment then maybe we can figure out how to officially put it in the build